src/main/java/net/mcreator/element/types/Block.java

```diff
@@ -77,7 +77,7 @@
 	public List<BoxEntry> boundingBoxes;
 
 	public String name;
-	public List<String> specialInfo;
+	public TextProcedure specialInformation;

```

Comment:
Either rename this back here or rename it in templates.

Reply:
Why?

Reply:
>2022-06-12-16:44:56[Test worker/ERROR][Template Generator] Failed to generate template: block.java.ftl
    freemarker.core.InvalidReferenceException: The following has evaluated to null or missing:
    ==> data.specialInfo  [in template "block.java.ftl" at line 310, column 40]

Reply:
Done, it's fixed